tests/statistics_test.sh

```diff
@@ -194,11 +196,39 @@ function month_statistics_Test
 function year_statistics_Test
 {
   local target_year="2019"
+  local year_data
   local msg="Currently, kw does not have any data for the requested year."
+  local line
 
-  ID=1
+  declare -a expected_cmd=(
+    "2020 summary"
```

Comment:
Use `'` instead of `"` around strings that don't have a bash variable inside quotes. I know that many parts of the code don't follow it for legacy reasons, but let's trying to change it. Replate these quotes in your patch.